### PR TITLE
Edit route/controller for organization repo list

### PR DESCRIPTION
**Pull request by albson**
Addressing issue #2

### CODE REVIEW

File: lib/git-controller.js [hunk 1]
```diff
@@ -10,4 +10,15 @@ async function getOrgUsers(org) {
...
+  let options = {
+    method: 'GET',
+    url: `https://api.github.com/orgs/${org}/repos`,
+    headers: {'user-agent': 'node.js'}
+  };
+  return await request(options);
+}
+
 export { getOrgUsers };
+export { getOrgProjects };
```
Comment (moizjv) [inline lib/git-controller.js]: this can go in `export { getOrgUsers, getOrgProjects };`

File: lib/routes.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import koa_router from "koa-router";
 import { getOrgUsers } from "./git-controller";
```
Comment (moizjv) [inline lib/routes.js]: `import * as gitController from './git-controller`;`

**Comment by moizjv** [comment]
please add test and see if you can configure travis CI

**Comment by albson** [comment]
@moizjv updated with suggestions you have made

**Pull request merged**